docs/the-guide/chapter-5-track-model-evolutions-with-dvc/dvc.lock

```diff
@@ -30,60 +30,60 @@ stages:
       size: 2490
     params:
       params.yaml:
-        featurize.max_features: 100
-        featurize.ngrams: 1
+        featurize.max_features: 200
+        featurize.ngrams: 2
```

Comment:
I've thought back at why this is updated, and the reason is that I've copied the checkpoints based on what I had locally, not on Git. This would allow anyone to start or compare their local code with the checkpoints.